assignments/arrays.js

```diff
@@ -63,32 +63,59 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i]['id'] === 33) {
+        console.log('Car 33 is a ' + inventory[i]['car_year'] + ' ' + inventory[i]['car_make'] + ' ' + inventory[i]['car_model'] + '.');
+    }
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar = inventory[inventory.length - 1];
+console.log(lastCar.car_make + ' ' + lastCar.car_model + '.');
 
```

Comment:
I highly recommend using template literals in the future instead of + ' ' +
it would look something like this
console.log(`lastCar.car_make  lastCar.car_model`)
let me know if you want more information on string template literals